### ISSUE TITLE
Dataset __getitem__ Wrapper (inside custom Dataset class)

### ISSUE THREAD

**Issue by Bempong-Sylvester-Obese** [Bempong-Sylvester-Obese/RSNA-Intracranial-Aneurysm-Detection#4]
Wrap the whole body in a try/except and provide a dummy tensor fallback. Replace original __getitem__ with the  class below:

`from pydicom.errors import InvalidDicomError

def __getitem__(self, idx):
    try:
        row = self.df.iloc[idx]
        series_id = row[Config.ID_COL]
        series_path = os.path.join(self.series_dir, series_id)
        
        # Load volume
        volume = self.processor.load_dicom_series(series_path)
        
        # Load REAL segmentation mask from data
        mask = self.load_segmentation_mask(series_id, volume.shape)
        
        # Get aneurysm presence label
        has_aneurysm = int(row[Config.TARGET_COL])
        
        # Transform
        data_dict = {'volume': volume}
        if self.transform:
            data_dict = self.transform(data_dict)
        
        volume_tensor = data_dict['volume'].unsqueeze(0)  # Add channel dim
        mask_tensor = torch.from_numpy(mask).unsqueeze(0)
        
        return {
            'volume': volume_tensor,
            'mask': mask_tensor,
            'has_aneurysm': torch.tensor(has_aneurysm, dtype=torch.float32),
            'series_id': series_id
        }

    except (InvalidDicomError, Exception) as e:
        bad_id = self.df.iloc[idx][Config.ID_COL]
        print(f"[Dataset] Skipping corrupt/failed sample {bad_id}: {e}")
        
        # return dummy tensors to keep DataLoader alive
        dummy_vol = torch.zeros((1, 128, 128, 128), dtype=torch.float32)
        dummy_mask = torch.zeros((1, 128, 128, 128), dtype=torch.float32)
        return {
            'volume': dummy_vol,
            'mask': dummy_mask,
            'has_aneurysm': torch.tensor(0.0, dtype=torch.float32),
            'series_id': "DUMMY_FAIL"
        }
`
